docs/guides/3_generating_proofs.md

```diff
@@ -0,0 +1,55 @@
+# Generating Proofs For Aligned
+
+## Table of contents
+
+- [SP1](#sp1)
+    - [Dependencies](#dependencies)
+    - [How to generate a proof](#how-to-generate-a-proof)
+    - [How to get the proof verified by Aligned](#how-to-get-the-proof-verified-by-aligned)
+
+## SP1
+
+### Dependencies
+
+This guide assumes that:
+
+- sp1 prover installed (instructions [here](https://succinctlabs.github.io/sp1/getting-started/install.html))
+- sp1 project to generate the proofs
+  (instructions [here](https://succinctlabs.github.io/sp1/generating-proofs/setup.html))
+- aligned installed (instructions [here](../introduction/1_getting_started.md#quickstart))
```

Comment:
`"(instructions"` parenthesis are not closed

Reply:
I think they are